sky/server/server.py

```diff
@@ -675,6 +675,19 @@ async def logs(
     )
 
 
+@app.post('/users')
+async def users(request: fastapi.Request,
+                users_body: payloads.UsersBody) -> None:
+    """Gets all users."""
+    executor.schedule_request(
+        request_id=request.state.request_id,
+        request_name='users',
+        request_body=users_body,
+        func=core.users,
+        schedule_type=requests_lib.ScheduleType.SHORT,
+    )
```

Comment:
For API's that does not involve any sky resource operations, we can just do it in the handler as coroutine to avoid the overhead of workers. `'/api/get'` handler is a good reference